src/modules/modifyTask.js

```diff
@@ -0,0 +1,13 @@
+import { toNumber } from 'lodash';
```

Comment:
Please avoid using libraries if you don't strictly need them (YAGNI principle). This could cause a bigger production bundle.

Kindly please consider using built-in functions (parseInt, parseFloat, Number) or adding a plus sing before a string